.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 ## Product Description
-<!-- For non-invisible changes, describe user-facing effects. -->
+<!-- For non-invisible changes, describe user-facing effects and include screenshots. -->
 
 ## Technical Summary
 <!--
-    Provide a link to the ticket or document which prompted this change,
-    Describe the rationale and design decisions.
+    Provide a link to the ticket, design document, and/or technical specification
```

Comment:
nit: link to the related ticket, design document and technical specification, whichever available.

Can phrase it better may be but same thought as above about not making things mandatory since not all of these are available for each PR

Reply:
I proposed a rewording in https://github.com/dimagi/commcare-hq/pull/35083, let me know what you think

Reply:
Looks great, thank you!